tests/packagedcode/data/opam/package/ocaml-variants.opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
```

Comment:
This looks like a bit like YAML... have you tries to parse it as YAML?

Reply:
I tried to parse it as YAML but I was not able to get success.

Reply:
Yes, we can handle triple quotes too, we just need to add a command like `if '"""' in line: ...`, something like this.